.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install poetry
+        poetry install
+    - name: Lint check
+      run: poetry run black --check .
```

Comment:
The lint step does not need to run against 3 versions of Python, just the latest is fine.

Note: this can be addressed in another PR

Reply:
OK this one I will leave for a future PR